strings-csharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringMethods.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text;
+
+namespace GenerateStreamFromStringInCSharp
+{
+    public static class GenerateStreamFromStringMethods
+    {
+        public static Stream
+        UseMemoryStreamAndStreamWriter(string sampleString, Encoding? encoding = null)
+        {
+            encoding ??= Encoding.UTF8;
+
+            var stream = new MemoryStream(encoding.GetByteCount(sampleString));
+            using var writer = new StreamWriter(stream, encoding, -1, true);
+            writer.Write(sampleString);
+            writer.Flush();
+            stream.Position = 0;
+
+            return stream;
+        }
+
+        public static Stream
+        UseGetBytesAndMemoryStream(string sampleString, Encoding? encoding = null)
```

Comment:
Instead of breaking this method into multiple lines, let's keep it in one line and change its name to be GetStreamWithGetBytes()
```suggestion
        public static Stream GetStreamWithGetBytes(string sampleString, Encoding? encoding = null)
```